### ISSUE TITLE
problems building (OR)AND query with field names

### ISSUE THREAD

**Issue by tonymillion** [scholrly/lucene-querybuilder#20]
I'm trying to build a query using the following code:
`q = Q(Q('class','comment') | Q('class','post')) & Q('text','things')`

I was expecting the output to be:
`(class:(post) OR class:(comment)) text:'things'`

instead I get:
`(class:(comment) class:(post) AND text:(things))`

Which is obviously wrong and doesn't work.

---

For reference a query of:
`Q( Q('a')|Q('b')) & Q('text','things')`

does actually output the correct query:
`((a OR b) AND text:(things))`

**Comment by wyc** [comment]
Yep, I'm getting the same. Also noticed that the `+` (must) operator does not encapsulate terms as expected:

```python
>>> q = Q('Mary had a little lamb') & +(Q('author', 'jimmy') | Q('author', 'jack'))
>>> str(q)
'("Mary had a little lamb" AND +author:(jimmy) author:(jack))'
```

Expected:

```python
'("Mary had a little lamb" AND +(author:(jimmy) OR author:(jack)))'
```

**Comment by ritiksoni00** [comment]
Hey @wyc , hope you are having good one!

can you tell me more about (in case you remember)
> + (must) operator does not encapsulate terms as expected:
